components/feature/recents/src/main/java/mozilla/components/feature/recents/RecentApp.kt

```diff
@@ -0,0 +1,22 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package mozilla.components.feature.recents
+
+interface RecentApp {
+    /**
+     * Unique ID of this recent app.
+     */
+    val id: Long
+
+    /**
+     * The name of the recent app.
+     */
+    val name: String
```

Comment:
Do we really want to save the name of the app? Isn't the package name enough? Maybe we could even use the package name as primary key?

Reply:
Yep, I now use the packageName as the unique PK, thanks!